site/app/controllers/MiscController.php

```diff
@@ -181,6 +179,8 @@ public function displayFile($dir = null, $path = null, $gradeable_id = null, $us
                 $this->core->getOutput()->renderOutput('Misc', 'displayCode', $file_type, $corrected_name, $contents);
             }
             else {
+                $this->core->getOutput()->useHeader(false);
+                $this->core->getOutput()->useFooter(false);
```

Comment:
What purpose did this change serve?  (Just to be clear, I'm not requesting changes here.  I just didn't see the reasoning behind it immediately and wanted to understand)

Reply:
I think I was thinking that the file was a raw file that was being outputted but that's not the case; I changed it to also use content_only here